src/runtime/components/CTA/CTA2.vue

```diff
@@ -299,11 +302,11 @@ const submitForm = (event) => {
             token,
           };
           axios
-            .post(config.API_BASE +"/api/send-contact-mail", formData)
+            .post(apiBase + "/api/send-contact-mail", formData)
```

Comment:
We should get whole `contactApiUrl` instead of `apiBase`, as it can be different for others